.github/workflows/platform-release.yaml

```diff
@@ -194,33 +226,108 @@ jobs:
           fi
           gh release create "${release_args[@]}"
 
-      - name: Checkout
-        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
-
       - name: Generate CLI documentation
         shell: bash
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TAG_NAME: ${{ inputs.release_tag || github.ref_name}}
         run: |
+          pushd kitops
```

Comment:
This is maybe a side effect of github actions hell, but what's changed to require us to enter this directory here?

Reply:
having two repositories checked out side-by-side:  kitops and homebrew-kitops